recipes/drmaa/run_test.py

```diff
@@ -0,0 +1,4 @@
+try:
+    import drmaa
```

Comment:
Makes sense.  I was not familiar with the details of libdrmaa, given the complexities I agree testing does not make sense.  Adding notes how to configure the package seems the best option.